doc/source/operations.rst

```diff
@@ -589,3 +589,123 @@ To avoid having to pass "-n openstack" all the time:
 .. code-block:: console
 
    kubectl config set-context $(kubectl config current-context) --namespace=openstack
+
+Cheat-sheet
+===========
+
+**Kubernetes useful commands**
+
+To get namespaces
+
+.. code-block:: console
+
+   kubectl get namespaces
+
+To get the cluster nodes and their status
+
+.. code-block:: console
+
+  kubectl get nodes -n <namespace>
+
+To list the pods are running in what node in an specific namespace
```

Comment:
Maybe change to

To list the running pods and their nodes in a specific namespace